Ryujinx.HLE/HOS/Services/Ssl/SslService/ISslContext.cs

```diff
@@ -1,7 +1,25 @@
+using Ryujinx.Common.Logging;
+using System;
+
 namespace Ryujinx.HLE.HOS.Services.Ssl.SslService
 {
     class ISslContext : IpcService
     {
-        public ISslContext() { }
+        public ISslContext(ServiceCtx context ) { }
+        
+        [Command(4)]
+        public ResultCode ImportServerPki(ServiceCtx context)
+        {
+            var CertificateFormat = context.RequestData.ReadInt32();
```

Comment:
Avoid overusing var:
```suggestion
            int certificateFormat = context.RequestData.ReadInt32();
```